src/models/SpecificationFile.ts

```diff
@@ -65,10 +108,21 @@ export async function nameType(name: string): Promise<string> {
     }
     return TYPE_CONTEXT_NAME;
   } catch (e) {
+    if (await isURL(name)) {return TYPE_URL_PATH;}
     return TYPE_CONTEXT_NAME;
   }
 }
 
+export async function isURL(urlpath: string): Promise<boolean> {
+  try {
+    const url = new URL(urlpath);
+    if (url.protocol === 'http:' || url.protocol === 'https:') {return true;}
```

Comment:
```suggestion
    return (url.protocol === 'http:' || url.protocol === 'https:');
```

This way you can get rid of the `return false;` in line 123 (not the one inside the catch).